gateway-service-impl/src/main/java/org/hypertrace/gateway/service/baseline/BaselineServiceQueryParser.java

```diff
@@ -130,9 +144,7 @@ public BaselineEntitiesResponse parseQueryResponse(
         new LinkedHashMap<>();
     while (resultSetChunkIterator.hasNext()) {
       ResultSetChunk chunk = resultSetChunkIterator.next();
-      if (LOG.isDebugEnabled()) {
-        LOG.debug("Received chunk: " + chunk.toString());
-      }
+      LOG.debug("Received chunk: {} ", chunk.toString());
```

Comment:
```suggestion
      LOG.debug("Received chunk: {} ", chunk);
```
This removes the need for the if because the expensive stringification only happens if the message is needed to be logged.

Reply:
done